### ISSUE TITLE
Add command-line query

### ISSUE THREAD

**Issue by gtfierro** [gtfierro/durandal#10]
Need the scriptable kind where we can add a query on the command line and redirect the output to a file

**Comment by gtfierro** [comment]
Implemented in e78e74a529adeb806f094e5d65c9fa5b6f3dc8dc and f2189dca1ffd24c4ce7f90d103349ecee02a6a3e

Still need to make sure that the output is fully JSON-serializable:

Currently we get output like

```json
{
  "_name": "velocityX",
  "path": "nvrnSE4pJe4ZMO3WQdb-EPi5iwuzmTVUpk6XNNRGYsc=/anemometer/data/mpa/ref_1_2/s.anemometer/98bed0134528465a/i.anemometerdata/signal/feed",
  "uuid": "c98b4045-b190-3586-8449-bb950bcd0a3d"
}{
  "_name": "velocityY",
  "path": "nvrnSE4pJe4ZMO3WQdb-EPi5iwuzmTVUpk6XNNRGYsc=/anemometer/data/mpa/ref_1_2/s.anemometer/98bed0134528465a/i.anemometerdata/signal/feed",
  "uuid": "ff5766f1-6195-3429-a1b3-526fb2608918"
}
```

in which there is no comma between the two printed structs. In order to make this tool useful (e.g. allow JSON tools to parse and traverse the results), this needs to be fixed

**Comment by gtfierro** [comment]
Fixed as of e95e4515947499151818b472d4c0d8899ed37201
